airbyte-integrations/connectors/destination-weaviate/destination_weaviate/spec.json

```diff
@@ -0,0 +1,60 @@
+{
+  "documentationUrl" : "https://docs.airbyte.com/integrations/destinations/weaviate",
+  "supported_destination_sync_modes" : [
+    "append"
```

Comment:
why is the connector no support full_refresh mode? can't you drop a class before doing any writes?

Reply:
You're spot on. I will add this by doing:
1. Save the existing schema
2. Delete the class
3. Recreate the class from existing schema

Reply:
done, please verify and resolve. I also added test cases for this.